### ISSUE TITLE
Syntax error in Exercise 2: 01 Data Preparation Cmd 57: f.floor not existing

### ISSUE THREAD

**Issue by striple66** [microsoft/MCW-Big-data-and-visualization#58]
Hi,
in Exercise 2: 01 Data Preparation Cmd 57, the first line of code has a wrong syntax issue:
`df = dfWeather.withColumn('Hour', f.floor(dfWeather['Time']/100))` throws an NameError: name 'f' is not defined. Instead, it must be:
`df = dfWeather.withColumn('Hour', F.floor(dfWeather['Time']/100))`. Either change the Cmd 57 line 1 or change the Cmd 3 line 5: `from pyspark.sql import functions as f`

**Comment by joelhulen** [comment]
I walked through the lab up to this point. The notebook I downloaded from the repo already had the correct syntax: `df = dfWeather.withColumn('Hour', F.floor(dfWeather['Time']/100))`. Did you download the notebooks from this repo directly in Databricks, as instructed? The link in the instructions is relative to the HoL document. I wonder if perhaps you are following steps from a forked repo or some other location?
